metricbeat/module/http/_meta/config.yml

```diff
@@ -8,3 +8,14 @@
   #method: "GET"
   #request.enabled: false
   #response.enabled: false
+
+- module: http
+  metricsets: ["server"]
+  host: "localhost"
+  port: "8080"
+  enabled: false
+#  paths:
+#    - path: "/foo"
+#      namespace: "foo"
+#      tags:
```

Comment:
I think it would make sense to change this behavior, use `fields` instead of tags, and put given fields in the final event, overriding any existing one